omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1988 @@
+********************
+Version requirements
+********************
+
+Summary of planned changes for OMERO 5.1 and 5.2
+================================================
+
+Criteria for what is considered to be supportable includes support
+both by its developers upstream and within each operating system
+distribution for the lifetime of the 5.1 release (including security
+support), and also upon our resources allocated to CI and testing. If
+we are not actively testing it, we cannot claim it is supported or
+functional.  Software components must be provided and supported either
+by a distribution or or the original developers.
```

Comment:
You have two 'or's here. Also the 1st sentence is a bit of a mouthful, what about starting "Criteria for what we support includes whether upstream support from developers and operating system distributions will be available for the lifetime of the…" etc, etc